addons/website_event_booth/controllers/event_booth.py

```diff
@@ -18,9 +18,10 @@ def event_booth_main(self, event, booth_category_id=False, booth_ids=False):
             raise Forbidden()
 
         booth_category_id = int(booth_category_id) if booth_category_id else False
+        seo_object = event.booth_menu_ids
```

Comment:
I think that for most cases here, we can pass on the `seo_object` as we use it only in one place and we don't gain much space either. So I would directly use the value except for the one you do a filter on.